src/crystal/system/unix/mime.cr

```diff
@@ -0,0 +1,24 @@
+module Crystal::System::MIME
+  MIME_SOURCES = {
```

Comment:
There are already files for OS specific implementations eg. `/src/crystal/system/unix|win32`.
Should these go in there own OS implementations?